FWCore/Framework/src/EventProcessor.cc

```diff
@@ -434,6 +434,8 @@ namespace edm {
         nStreams = nThreads;
       }
     }
+    // PG: Log the number of streams
+    edm::LogInfo("StreamSetup") <<"setting # streams "<<nStreams;
```

Comment:
Only print number of streams if the option was explicitly set. That would be moving this to be between line 435 and 436.